
The usual kind of user-space application fix.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.8.1/include/linux/videodev2.h.orig	2004-09-15 15:45:53.633962050 +0200
+++ linux-2.6.8.1/include/linux/videodev2.h	2004-09-15 15:45:47.348967172 +0200
@@ -17,6 +17,8 @@
 #include <linux/time.h> /* need struct timeval */
 #endif
 
+#include <linux/compiler.h>
+
 /*
  *	M I S C E L L A N E O U S
  */
